ckanext/gla/templates/admin/index.html

```diff
@@ -0,0 +1,36 @@
+{% ckan_extends %}
+
+{% block secondary_content %}
+<div class="module module-narrow module-shallow">
+    <h2 class="module-heading">
+        <i class="fa fa-info-circle"></i>
+        {{ _('Administer CKAN') }}
+    </h2>
+    <div class="module-content">
+
+        {% set docs_url = "http://docs.ckan.org/en/{0}/sysadmin-guide.html".format(g.ckan_doc_version) %}
+        {% trans %}
+        <p>As a sysadmin user you have full control over this CKAN instance. Proceed with care!</p>
+        <p>For guidance on using sysadmin features, see the CKAN  <a href="{{ docs_url }}" target="_blank" rel="noreferrer">sysadmin guide</a></p>
+        {% endtrans %}
+    </div>
+    <div class="module-content">
+        <p>System administrators can download a log of all user searches, and the results that they selected
+            as a csv file <a id="downloadlink"  data-module="gla_search_result_log_download">here</a>.</p>
```

Comment:
Similar comment to `package_item.html` - I don't think the `data-module` is doing anything here.
```suggestion
            as a csv file <a id="downloadlink">here</a>.</p>
```